src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -244,13 +243,11 @@ public StreamingResponseBody streamEvents(
                         .build();
 
                 // acquire connection slots to limit the number of simultaneous connections from one client
-                if (featureToggleService.isFeatureEnabled(LIMIT_CONSUMERS_NUMBER)) {
```

Comment:
@ferbncode 
Please check the actual status (ON/OFF) on live/staging systems of each feature-toggle you remove. 
For example, this one - LIMIT_CONSUMERS_NUMBER is currently OFF on all environments, but after your changes it will become always ON.

Reply:
@ferbncode in order to check if a feature toggle is turned ON/OFF you can use the following endpoints:

```
TOKEN=`zign token`
export NAKADI_URL=https://nakadi-staging.aruha-test.zalan.do
curl -v -XGET -H "Authorization: Bearer ${TOKEN}" $NAKADI_URL/settings/features | jq
```